shared-scripts.sh

```diff
@@ -3,15 +3,29 @@
 # set exit on error to true
 set -e
 
+# The flags address the issue here: https://github.com/boto/botocore/issues/1716
+export MSYS_NO_PATHCONV=1
+export MSYS2_ARG_CONV_EXCL="*"
+
 # storeCache <local path> <cache location>
 function storeCache {
   localPath="$1"
   alias="$2"
+  environment="$3"
   s3Path="s3://$CACHE_BUCKET_NAME/$CODEBUILD_SOURCE_VERSION/$alias"
   echo "Writing cache folder $alias to $s3Path"
   # zip contents and upload to s3
-  if ! (cd $localPath && tar cz . | aws s3 cp - $s3Path); then
-      echo "Something went wrong storing the cache folder $alias."
+  errorMessage="Something went wrong storing the cache folder $alias."
```

Comment:
nit: add "continuing anyway" clause to your error message as below, since it looks like an error storing the cache isn't fatal

Reply:
It isn't fatal, but it probably should be. If we fail to store the cache the test step will fail later. I'll update the error message now, and follow up with a change to fail the build.